base/math.jl

```diff
@@ -622,6 +622,8 @@ The article is available online at ArXiv at the link
 
 Compute the hypotenuse ``\\sqrt{\\sum |x_i|^2}`` avoiding overflow and underflow.
 
+See also `norm` in the [`LinearAlgebra`](@ref man-linalg) standard library.
```

Comment:
```suggestion
See also [`norm`](@ref) in the [`LinearAlgebra`](@ref man-linalg) standard library.
```

Reply:
I assumed this would fail, since norm is in stdlib/LinearAlgebra, but might be wrong.

Reply:
Now it fails like so:
```
    [ Info: CrossReferences: building cross-references.
  | ┌ Warning: no doc found for reference '[`norm`](@ref)' in src/base/math.md.
  | └ @ Documenter.CrossReferences /cache/build/amdci4-1/julialang/julia-master/doc/deps/packages/Documenter/f5jts/src/CrossReferences.jl:160
  | [ Info: CheckDocument: running document checks.
  | [ Info: Populate: populating indices.
  | ERROR: LoadError: `makedocs` encountered an error. Terminating build
```
If there is a way to refer to standard library functions, though, that would often be nice.

Reply:
I thought that would just work!

Reply:
What if you call it `LinearAlgebra.norm`?

Reply:
I don't think I tried that here. But I have failed in the past to find a way to do that, e.g. in `help?> Threads.nthreads`.